website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,189 @@
+
+The footer for the Single component includes a total count, which is customizable to fit the context.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
+
+##### Multiple
+
+The footer for the Multiple component includes buttons to modify the options and a counter of the number of options selected out of the total.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-multiple-variant-footer.png)
+
+The “Show selected” button displays only the selected options in the list. When activated, it toggles to “Show all.”
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png)
+
+If “Show selected” is used when no options have been selected, a message will appear notifying the user.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png)
+
+The “Clear selected” button clears all selections and reverts the Super Select to its default/deselected state. This button is hidden if no options are selected.
+
+![Flow showing the behavior of the clear selected button and how it reverts the Super Select's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png)
+
+##### Footer with custom content
+
+The footer can contain custom content instead of the default content. This is helpful in maintaining the standard appearance of the footer container while allowing for custom usage.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
+
+## List
+
+### Positioning
+
+Lists can be positioned to the left or right, as well as above or below the trigger as necessary to fit within the UI. These options are only available when `matchTriggerWidth` is set to `false`.
+
+![Showing the different list positions (left, right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+By default, the width of the list matches the trigger width.
+
+![Showing the list's width matching the trigger's](/assets/components/form/super-select/super-select-list-width-matching.png)
+
+#### Auto width
+
+If the list width is set to “auto,” it will match the width of the longest list item. This means it may be either narrower or wider than the trigger width accordingly. A `max-width` can also be defined for the list.
```

Comment:
[suggestion] What do you think of this? Seems to read a bit better to me

```suggestion
If the list width is set to “auto" it will match the width of the longest list item, which may result in the list being narrower or wider than the trigger width. A `max-width` can also be defined for the list.
```

Reply:
Agreed.